### ISSUE TITLE
BeautifulSoup with "xml" gets ofx file ("The ofx file is empty!")

### ISSUE THREAD

**Issue by sreeshas** [jseutter/ofxparse#107]
This is similar to issue #76 

In ofxparse.py, 

```
def soup_maker(fh):
    skip_headers(fh)
    try:
        from bs4 import BeautifulSoup
        soup = BeautifulSoup(fh, "xml")
        for tag in soup.findAll():
            tag.name = tag.name.lower()
    except ImportError:
        from BeautifulSoup import BeautifulStoneSoup
        soup = BeautifulStoneSoup(fh)
    return soup
```

"xml" parser returns "The ofx file is empty!". 
If I choose "lxml" instead of "xml", the tests fail for suncorp.ofx.  This is because suncorp.ofx has CDATA which cannot be parsed correctly by lxml in current condition. 

If i choose "html.parser", everything works as expected. 

I have tested the above behavior on Mac OS X with Python 2.7.

**Comment by shawkinaw** [comment]
I ran into a similar issue. Had the same result (empty OFX file) and same solution (use `lxml` parser). I think I could probably also fix by downgrading to BeautifulSoup 3, but haven't tried that.

In my case, my OFX files don't have a top-level `<OFX>` tag containing the contents, but rather starts right into `<SIGNONMSGSRSV1>` after the headers. There actually is an `<OFX>` tag, but it's the very last thing in the file.

If it would be helpful, I can sanitize an example file and pass along.

BTW, thanks so much for this library! I'm especially happy that you've updated it to handle all the data members I need (in particular, the total amount of investment transactions.) Funny that this was just updated, and I just decided to finally fix that in my accounting scripts. Sorry for the non-sequitur, just wanted to express my gratitude for this project.
